docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+services:
+  cache:
+    build: .
+    ports:
+      - "80:80"
+      - "8443:8443"
+    volumes:
+      - /data/cache:/cache
+      - /data/logs:/var/log/nginx
+      - /data/epic-certs:/etc/nginx/ssl/epic/
```

Comment:
@gammasts can you comment this epic line from the PR? nginx won't start if the certs are missing and given how incredibly niche this is it's not a sane default to expect everyone to generate certs they won't use